hail/python/hail/docs/hailpedia/table.rst

```diff
@@ -100,10 +99,10 @@ special characters in it. The Python type of each attribute is an
 :class:`.Expression` that also contains context about its type and source, in
 this case a row field of table `ht`.
 
-    >>> ht
+    >>> ht  # doctest: +NOTEST
     <hail.table.Table at 0x110791a20>
 
-    >>> ht.ID
+    >>> ht.ID  # doctest: +NOTEST
```

Comment:
we could leave this one right?

Reply:
yeah. I mostly just filled these in in places I didn't especially care about (str representations of objects, etc)